python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -890,6 +914,37 @@ def _fast_slow_function_call(func: Callable, /, *args, **kwargs) -> Any:
                 # try slow path
                 raise Exception()
             fast = True
+            if cudf_pandas_debugging:
+                try:
+                    with nvtx.annotate(
+                        "EXECUTE_SLOW_DEBUG",
+                        color=_CUDF_PANDAS_NVTX_COLORS["EXECUTE_SLOW"],
+                        domain="cudf_pandas",
+                    ):
+                        slow_args, slow_kwargs = (
+                            _slow_arg(args),
+                            _slow_arg(kwargs),
+                        )
+                        with disable_module_accelerator():
+                            slow_result = func(*slow_args, **slow_kwargs)
+                except Exception as e:
+                    warnings.warn(
+                        "The result from pandas could not be computed. "
+                        f"The exception was {e}."
+                    )
+                else:
+                    try:
+                        _assert_fast_slow_eq(result, slow_result, **kwargs)
```

Comment:
```suggestion
                        _assert_fast_slow_eq(result, slow_result)
```

I think these `kwargs` are only intended for `func` and not for asserting equality.

Reply:
I'm also using them in `_assert_slow_function_eq` to get the type of assert function to use. @bdice wanted this to be configurable too. I'm not sure if that's the best way to do it though.

Reply:
@galipremsagar is correct here. I meant that I wanted the _choice of function_ to be configurable, not necessarily the arguments you pass to it. This is again in the spirit of keeping the proxy machinery generic -- we may not want to hardcode "use the cudf equality testing function" because that isn't necessarily the right choice of function if we wanted to proxy another pair of modules like cupy and numpy.